src/PerfView/Extensibility.cs

```diff
@@ -994,6 +994,36 @@ public void LookupSymbolsForModule(string simpleModuleName, SymbolReaderOptions
             }
         }
 
+        /// <summary>
+        /// Saves the stacks as a CSV in the same format as it would appear in the CPUStacks GUI
+        /// </summary>
+        /// <param name="outputFileName"> The file name the data will be written to </param>
+        public void SaveAsCsvByName(string outputFileName)
+        {
+            if (File.Exists(outputFileName))
+            {
+                File.Delete(outputFileName);
+            }
+            using (var csvFile = File.CreateText(outputFileName))
+            {
+                csvFile.Write("Name,Exc,Exc%,Inc,Inc%,Fold,First,Last\r\n");
+                var callTree = ByName;
+                foreach (var callTreeNode in callTree)
+                {
+                    var frameUpdated = callTreeNode.Name.Replace(",", ";");
+                    csvFile.Write("{0},{1},{2},{3},{4},{5},{6},{7}\r\n",
```

Comment:
NIT: I have been bit by this before so, thought would mention that string interpolation reduces the risk of misordering the columns especially if you add / change them. 

```csharp
csvFile.Write($"{frameUpdated},{callTreeNode.ExclusiveMetric},{callTreeNode.ExclusiveMetricPercent},
{callTreeNode.InclusiveCount},{callTreeNode.InclusiveMetricPercent},{callTreeNode.ExclusiveFoldedMetric},{callTreeNode.FirstTimeRelativeMSec},{callTreeNode.LastTimeRelativeMSec}\r\n");

Reply:
I think this is a good call.  I realized how hard it was to view the columns if they are formatted like this.  Will fix this so its easier to read and maintain in the future